medusa/processTV.py

```diff
@@ -489,54 +487,55 @@ def unRAR(path, rarFiles, force, result):  # pylint: disable=too-many-branches,t
     return unpacked_files
 
 
-def already_postprocessed(dirName, videofile, force, result):  # pylint: disable=unused-argument
+def already_postprocessed(dir_name, video_file, force, result):
     """
-    Check if we already post processed a file
+    Check if we already post processed a file.
 
-    :param dirName: Directory a file resides in
-    :param videofile: File name
+    :param dir_name: Directory a file resides in
+    :param video_file: File name
     :param force: Force checking when already checking (currently unused)
     :param result: True if file is already postprocessed, False if not
     :return:
     """
     if force:
         return False
 
-    # Avoid processing the same dir again if we use a process method <> move
+    file_path = os.path.join(dir_name, video_file)
+    file_size = os.path.getsize(file_path)
```

Comment:
Makes the check useless. The whole point is to make sure the file is the same by checking the size.